beta/src/styles/sandpack.css

```diff
@@ -2,289 +2,268 @@
  * Copyright (c) Facebook, Inc. and its affiliates.
```

Comment:
Most of the changes from this PR are on this file. Here are the highlights of these changes:
- Removed unnecessaries `!important`;
- Tried to reuse tailwind variables instead hardcoded colors;
- Addressed breaking changes due to the new Sandpack version;
- Reorganized and regrouped styles by categories;
- Kept the "Read more" section as it was, there was no need to changes it;